platform/ui/src/components/userPreferencesModal/AboutModal.js

```diff
@@ -112,6 +126,7 @@ class AboutModal extends Component {
             {` `}
             <a
               className="btn btn-default"
+              data-cy="btn-more-details"
```

Comment:
Similarly, for these buttons, we can probably look for them by scoping our query to the modal (`data-cy="about-modal"`), and then just verifying that the text content of the button exists anywhere in the modal (contains text).

This allows us to scope where we're looking, but allows for things to move around, without breaking our tests.